project/ReleaseExtras.scala

```diff
@@ -0,0 +1,65 @@
+import sbt._
+import sbt.Keys._
+import sbt.Package._
+
+import sbtrelease._
+import sbtrelease.ReleasePlugin.ReleaseKeys._
+import sbtrelease.ReleaseStateTransformations._
+import sbtrelease.Utilities._
+
+object ReleaseExtras {
+  object ReleaseExtrasKeys {
+    val releaseBranchName = taskKey[String]("The name of the branch")
+  }
+
+  import ReleaseExtrasKeys._
+
+  implicit class RichGit(git: Git) {
+    def checkout(name: String): ProcessBuilder = git.cmd("checkout", name)
+
+    def checkoutNew(name: String, from: String, force: Boolean = false): ProcessBuilder =
+      git.cmd("checkout", if (force) "-B" else "-b", name, from)
+
+    def pushBranch(branch: String, remote: String): ProcessBuilder =
+      git.cmd("push", "-u", remote, branch)
+  }
+
+  private def git(st: State): Git = {
+    st.extract.get(versionControlSystem).collect {
+      case git: Git => git
+    }.getOrElse(sys.error("Aborting release. Working directory is not a repository of a Git."))
+  }
+
+  /**  This release step involves following actions:
+    *  - create version branch from current branch setting same remote
+    *  - checkout this branch and push it to remote
```

Comment:
Could you write 'checkout version branch and push it to remote' to make it more explicit.